charts/opennotificaties/templates/secret.yaml

```diff
@@ -16,8 +16,10 @@ stringData:
   {{- end }}
   {{- if .Values.tags.redis }}
   CELERY_RESULT_BACKEND: {{ printf "redis://%s-master:6379/1" (include "common.names.fullname" .Subcharts.redis) | quote }}
+  CELERY_RESULT_EXPIRES: {{ printf "3600" (include "common.names.fullname" .Subcharts.redis) | quote }}
```

Comment:
Just adding this envvar won't be enough, it also needs to be defined as a Django setting. Could you add it to OAF?